editor/plugins/spatial_editor_plugin.h

```diff
@@ -837,12 +837,11 @@ class EditorSpatialGizmoPlugin : public Resource {
 	static const int HIDDEN = 1;
 	static const int ON_TOP = 2;
 
-private:
+protected:
```

Comment:
Only `current_gizmos` needs to be `protected`, no?
But it doesn't hurt per se to also include the other two if you think that's better.

Reply:
Yes, I figured at some point I will need to access the other members form a derivative class.